uber/templates/regform.html

```diff
@@ -161,17 +161,17 @@
         </div>
         {% popup_link "../static_views/affiliates.html" "What's an affiliate?" %}
     </div>
-{% endif %}
 
```

Comment:
Oh no, this was my fault - I added the shirt size option in another PR because I thought it wasn't being displayed on the admin form.

Before we merge this, I'd like to undo the change you made here and remove my shirt row on the admin form - this is the better solution.